collectors/nvd/tests/conftest.py

```diff
@@ -11,6 +11,19 @@ def enable_db_access_for_all_tests(db):
     pass
 
 
+@pytest.fixture(autouse=True)
+def enable_env_vars(monkeypatch) -> None:
+    import apps.bbsync.mixins as bbsync_mixins
+    import apps.taskman.mixins as taskman_mixins
+    from collectors.nvd import collectors
+    from osidb import models
+
+    monkeypatch.setattr(bbsync_mixins, "SYNC_TO_BZ", True)
+    monkeypatch.setattr(taskman_mixins, "JIRA_TASKMAN_AUTO_SYNC_FLAW", True)
+    monkeypatch.setattr(models, "JIRA_TASKMAN_AUTO_SYNC_FLAW", True)
+    monkeypatch.setattr(collectors, "JIRA_AUTH_TOKEN", "SECRET")
```

Comment:
I will check this, thanks!

Reply:
It seems to be necessary. When I remove `monkeypatch.setattr(collectors, "JIRA_AUTH_TOKEN", "SECRET")` and `JIRA_AUTH_TOKEN` from my `.env`, tests are failing. But I check it via CI too.

Reply:
@costaconrado CI is failing when `JIRA_AUTH_TOKEN` is not set because CI does not have the `.env` file. When you want to rewrite a cassette, you usually just replace `SECRET` in e.g. `HTTP_BUGZILLA_API_KEY="SECRET"` by your current token. You would do the same in this case. I think that the `.env` file is ignored when rewriting a cassette. Does it make sense?

Reply:
it makes sense, LGTM.